doc/modules/changes.h

```diff
@@ -7,6 +7,11 @@
  *
  * <ol>
  *
+ * <li> New: There is now a mesh refinement criterion based on
+ * refining near a user-specified list of boundary indicators.
+ * <br>
```

Comment:
Can you reference the plugin by name here, so people don't have to go search for it throughout the manual?